mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule BambooSmtp.Mixfile do
       {:earmark, ">= 1.0.3", only: :dev},
       {:ex_doc, "~> 0.14.5", only: :dev},
       {:excoveralls, "~> 0.6.0", only: :test},
-      {:gen_smtp, "~> 0.11.0"},
+      {:gen_smtp, github: "Vagabond/gen_smtp", branch: "master"},
```

Comment:
Great! If they don't want to release something yet, we still have the option to reference a specific SHA. It will work and it will ensure they can't introduce breaking changes.

```elixir
{: gen_smtp, github: "Vagabond/gen_smtp", ref: "2c02d357b6cdedf9adcfdc49f6b250c5eb7203e2"}
```

But first, let see what they think about a new release. It would be a better solution :)